src/main/java/org/elasticsearch/cluster/block/ClusterBlockLevel.java

```diff
@@ -44,13 +55,35 @@ public int id() {
         return this.id;
     }
 
-    public static ClusterBlockLevel fromId(int id) {
+    /**
+     * Returns the ClusterBlockLevel's id according to a given version, this to ensure backward compatibility.
+     *
+     * @param version the version
+     * @return the ClusterBlockLevel's id
+     */
+    public int toId(Version version) {
+        //nocommit Update version number according to the branch it is merged to
+        if ((version != null) && (version.onOrBefore(Version.V_1_5_0))) {
```

Comment:
the version null check is a bit scary, when can the version be null? I would rather add an assert, so we catch it if it happens